src/MSBuild/LiveLogger/LiveLogger.cs

```diff
@@ -468,6 +473,12 @@ private void MessageRaised(object sender, BuildMessageEventArgs e)
                     _projects.TryGetValue(new ProjectContext(buildEventContext), out Project? project))
                 {
                     ReadOnlyMemory<char> outputPath = e.Message.AsMemory().Slice(index + 4);
+
+                    if (outputPath.Span.Slice(0, _initialWorkingDirectory.Length).SequenceEqual(_initialWorkingDirectory.AsSpan()))
+                    {
+                        outputPath = outputPath.Slice(_initialWorkingDirectory.Length + 1);
+                    }
```

Comment:
This should be done later when printing the link so that we can use the full absolute path for link target and the relative path for the text. Windows doesn't know how to open a relative link:

![image](https://user-images.githubusercontent.com/12206368/233922071-927d94bf-433e-4916-b563-8738396fc53d.png)

Reply:
Nice catch, thanks.